source/jormungandr/jormungandr/interfaces/v1/Schedules.py

```diff
@@ -109,6 +112,11 @@ def __init__(self, endpoint):
 
     def get(self, uri=None, region=None, lon=None, lat=None):
         args = self.parsers["get"].parse_args()
+
+        # for retrocompatibility purpose
+        for forbid_id in args.get('forbidden_id[]', []):
+            args.get('forbidden_uris[]', []).append(forbid_id)
```

Comment:
minor: you can do `args['forbidden_uris[]']` (and same above) because I think flask add by default all params in args

It's better because if it is not a valid flask argument you'll get an error